src/attribute.rs

```diff
@@ -13,17 +13,17 @@ pub(crate) enum Attribute<'src> {
   Doc(Option<StringLiteral<'src>>),
   Extension(StringLiteral<'src>),
   Group(StringLiteral<'src>),
-  Linux,
-  Macos,
+  Linux { inverted: bool },
```

Comment:
I think we should probably use:

```suggestion
  Linux { enabled: bool },
```

And invert the meaning of the boolean, to make the downstream code make more sense.